src/evergreen/evg_config.rs

```diff
@@ -44,8 +45,12 @@ impl EvgConfigService for EvgProjectConfig {
     }
 
     /// Get a map of task name to task definitions.
-    fn get_task_def_map(&self) -> HashMap<String, &EvgTask> {
-        self.evg_project.task_def_map()
+    fn get_task_def_map(&self) -> HashMap<String, EvgTask> {
```

Comment:
Since we will now be sharing this among threads, it was easier to take ownership of the `EvgTask` rather than having a reference to it.